knockknock/slack_sender.py

```diff
@@ -50,60 +55,174 @@ def wrapper_sender(*args, **kwargs):
                 master_process = True
 
             if master_process:
-                contents = ['Your training has started 🎬',
-                            'Machine name: %s' % host_name,
-                            'Main call: %s' % func_name,
-                            'Starting date: %s' % start_time.strftime(DATE_FORMAT)]
-                contents.append(' '.join(user_mentions))
-                dump['text'] = '\n'.join(contents)
-                dump['icon_emoji'] = ':clapper:'
+                notification = "Your training has started! 🎬"
+                if user_mentions:
+                    notification = _add_mentions(notification)
+
+                dump["blocks"] = _starting_message(
+                    func_name, host_name, notification, start_time
+                )
+                dump["text"] = notification
```

Comment:
Do you need the notification here too? It seems to me that it's already contained in the `block` field.

Reply:
Yes, removing it from here means that the message text will not display in desktop notifications